HHS/Head-Start-TTADP
# [TTAHUB-2060] Move the TTA provided from one page to another on the TR session form

## Description of change
- TTA Provided appears in the Session summary section
- TTA Provided field sits between Resource file upload and Support type fields to match what's in the AR

## How to test

Confirm that the field has been moved, appears in the right place, and that the session form still functions as expected

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2060


## Checklists

### Every PR



### Production Deploy


### After merge/deploy
